packages/app-admin/src/components/AppInstaller/AppInstaller.tsx

```diff
@@ -22,10 +22,15 @@ import { config as appConfig } from "@webiny/app/config";
 
 export const AppInstaller: React.FC = ({ children }) => {
     const tenantId = localStorage.get("webiny_tenant") || "root";
-
     const lsKey = `webiny_installation_${tenantId}`;
     const wbyVersion = appConfig.getKey("WEBINY_VERSION", process.env.REACT_APP_WEBINY_VERSION);
     const isRootTenant = tenantId === "root";
+    /* Cypress doesn't work with cross domains because of
+    * security issues. This setting allows us to avoid rendering
+    * the <iframe> without disabling chrome web security
+    * reference: https://docs.cypress.io/guides/guides/web-security#Insecure-Content 
+    */
```

Comment:
```suggestion

    /* 
    * This flag allows us to avoid rendering the <iframe> when the app is tested with Cypress
    * (Cypress doesn't work with cross domains because of security-related implications).
    * @see https://docs.cypress.io/guides/guides/web-security#Insecure-Content 
    */
```